esmf_regrid/__init__.py

```diff
@@ -7,6 +7,8 @@
     except ImportError:
         raise exc
 
+# constants is used within schemes, so needs to imported first
+from ._constants import *
 from .schemes import *
```

Comment:
The adoption of the `from <x> import *` pattern is generally discouraged, particularly if the source module (`<x>`) doesn't have a `__all__` defined.

There is plenty blurb out there backing up this argument e.g., see
- https://stackoverflow.com/questions/2360724/what-exactly-does-import-import
- https://stackoverflow.com/questions/44834/what-does-all-mean-in-python